### ISSUE TITLE
Delay app measurement doesn't start measurements after MobileAds.Inizialize() or ad load

### ISSUE THREAD

**Issue by RaffaLinux** [googleads/googleads-mobile-unity#1322]
### Step 1: Describe your environment

  * Unity version: 2019.3.2f1
  * Google Mobile Ads Unity plugin version: 5.1.0
  * Platform: Android
  * Platform OS version_All Android versions
  * Any specific devices issue occurs on: All devices
  * Mediation ad networks used, and their versions: None

#### Steps to reproduce:

I enabled the app measurement delay to comply with GDPR regulation and ask my user for consent before starting measurement. I initialize MobileAds in my Start function. 
I got tens of users but my user metrics are still 0 ( i also made sure to follow the troubleshooting on the Admob documentation). If i disable the delay app measurement option i start to receive user metrics correctly.
In the Admob guide (and in the Unity tooltip)  is said that the delay app measurement option is supposed to enable measurements after i initialize MobileAds but it's not working as supposed for me.

**Comment by stowy** [comment]
Hi @RaffaLinux thanks for reporting this, it sounds like an issue related to the Android SDK. Can you confirm that you are using the MobileAds.Initialize((initStatus) => {}) call and not the call that takes an App ID?

**Comment by RaffaLinux** [comment]
> Hi @RaffaLinux thanks for reporting this, it sounds like an issue related to the Android SDK. Can you confirm that you are using the MobileAds.Initialize((initStatus) => {}) call and not the call that takes an App ID?

Yes, I'm using MobileAds.Initialize((initStatus) => {})

**Comment by kunny** [comment]
Since there were no recent activities for this issue, we are therefore reluctantly going to close this bug for now. If you are still experiencing the problem, please file a new issue with the same description, what happens and system / [network tracing](https://developers.google.com/admob/android/network-tracing) / [Charles](https://developers.google.com/admob/android/charles) logs.

All system setups can be slightly different, so it's always better to open new issues and reference the related ones. Thanks for your contribution again!
